executor/README.md

```diff
@@ -89,6 +89,38 @@
             - name: FORCE
               value: "true"
     ```
+    
+### How to Override Executor and Exporter image
+
+-   To override the executor and exporter image variables , perform one of the below steps before applying the manifest:
```

Comment:
Something in the lines of: 

To provide a chaos executor (runner) & exporter (monitor) of choice, provide the image names (in the form `registry/user/image:tag`) in the chaosEngine as shown below. The defaults at this point correspond to the litmuschaos CI images in dockerhub:

- Executor: litmuschaos/ansible-runner:ci
- Monitor: litmuschaos/chaos-exporter:ci

Reply:
let's not say `one of the below` as there may be need for both.